include/imu_ros2/iio_wrapper.h

```diff
@@ -204,38 +204,38 @@ class IIOWrapper {
     static struct iio_channel *m_channel_temp;
     static struct iio_channel *m_channel_count;
 
-    static float m_fvalScaleAccel_x;
-    static float m_fvalScaleAccel_y;
-    static float m_fvalScaleAccel_z;
+    static double m_fvalScaleAccel_x;
+    static double m_fvalScaleAccel_y;
+    static double m_fvalScaleAccel_z;
 
-    static float m_fvalScaleAngvel_x;
-    static float m_fvalScaleAngvel_y;
-    static float m_fvalScaleAngvel_z;
+    static double m_fvalScaleAngvel_x;
+    static double m_fvalScaleAngvel_y;
+    static double m_fvalScaleAngvel_z;
 
-    static float m_fvalScaleRot_x;
-    static float m_fvalScaleRot_y;
-    static float m_fvalScaleRot_z;
+    static double m_fvalScaleRot_x;
+    static double m_fvalScaleRot_y;
+    static double m_fvalScaleRot_z;
 
-    static float m_fvalScaleVelocity_x;
-    static float m_fvalScaleVelocity_y;
-    static float m_fvalScaleVelocity_z;
+    static double m_fvalScaleVelocity_x;
+    static double m_fvalScaleVelocity_y;
+    static double m_fvalScaleVelocity_z;
 
-    static float m_fvalScaleTemp;
+    static double m_fvalScaleTemp;
```

Comment:
I am wondering if the fval naming should also be changed here.
also, below there are some double variables which do not contain the 'fval' in their name. it is better to have consistency between variable names